supertokens_python/recipe/passwordless/recipe.py

```diff
@@ -81,6 +85,9 @@ def __init__(self, recipe_id: str, app_info: AppInfo, contact_config: ContactCon
         else:
             self.email_delivery = email_delivery_ingredient
 
+        sms_delivery_ingredient = ingredients.sms_delivery if ingredients is not None else None
```

Comment:
I think ingredients is always not None right?

Reply:
Yup. Directly using `sms_delivery_ingredient = ingredients.sms_delivery` now.